modules/brainbrowser/js/brainbrowser.loris.js

```diff
@@ -836,7 +844,7 @@ $(function() {
     /////////////////////
     // Load the volumes.
     /////////////////////
-    request.then(function(filename) {
+    Promise.all([...promises]).then(() => {
```

Comment:
Is `[...promises]` a js idiom I'm not familiar with? Why not just `promises`?

Reply:
@driusan the `...` is the spread operator. See [here about it](https://dev.to/sagar/three-dots---in-javascript-26ci).
Insead of doing something like this:
```
Promise.all([Promise1, Promise2, Promise3])
 .then(result) => {
   console.log(result)
 })
 .catch(error => console.log(`Error in promises ${error}`))
```
I thought to use the spread operator. I'm not sure if it will work with just promises.

Reply:
I know what the spread operator does, but you're spreading out the results of an array and then putting the result back into a new array.

Reply:
@driusan lol you're right.

Reply:
@driusan @maltheism fixed now